src/services/github.test.ts

```diff
@@ -315,15 +315,24 @@ describe('GithubService', () => {
         } as any as Octokit;
 
         const service = new GithubService(mocktokit);
-        const criteria = {
+        const criteria1 = {
             search_type: 'code',
             search_query: 'org:testOrg path:/ filename:package.json in:path'
         };
 
-        const result = await service.getActiveReposForSearchTypeAndQuery(criteria);
+        const criteria2 = {
+            search_query: 'org:testOrg path:/ filename:package.json in:path'
```

Comment:
Should one of these include a property `search_type` set to `""` or some falsey value?

Reply:
We could, but it shouldn't be necessary. It's an optional parameter and when undefined, it hits the default case. In fact, if `search_type` is set to "" or a falsy value, the code will error due to https://github.com/NerdWalletOSS/shepherd/pull/377/files/5ad0285e1d7b7e12f1dc57cdf63ba23b82772902..982c25c27099f41d87a347f7a2345c81bf825aa6#diff-a9d5c72f5483cdb150974d030822fe95aeeba82f63bf72ae4b2c929fa7e50fb5R111. There's also a test that covers a bad value being passed: https://github.com/NerdWalletOSS/shepherd/pull/377/files/5ad0285e1d7b7e12f1dc57cdf63ba23b82772902..982c25c27099f41d87a347f7a2345c81bf825aa6#diff-8cde3a22acd42e1d42cdd48cb278b9902dce34a3f629dcb790841f2f8154f9b4R247.

Reply:
Makes sense.